scenarios/setup.xml

```diff
@@ -48,6 +48,11 @@
               requires="Onos-Wait-For-Ready"
               exec="docker-log-grep ${ODI} 'New pipeconf registered: org.stratumproject.fabric-upf.bmv2'"/>
 
+        <!-- BUG in onos-cfg, we need to specify POST and not post in the curl to do an actual post -->
```

Comment:
Can you override `onos-cfg` by placing it under `scenarios/bin`?

Reply:
I tried, onos-cfg depends on other tools/script. I changed it into the ONOS codebase: https://gerrit.onosproject.org/c/onos/+/25312

Reply:
Merged but this will require publishing artifacts to mvn central. I'm ok merging as-is for now. We can address the FIXME later.

Reply:
Yes, we also need to wait for a release, because `onos-test` package is published only on releases.